packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -117,22 +120,40 @@ function EnableNotification({
           }`}
         </span>
       )}
-      <p className="mt-2 w-full text-theme-label-tertiary tablet:w-3/5">
-        {acceptedJustNow ? (
-          <>
-            Changing your{' '}
-            <a
-              className="underline hover:no-underline"
-              href={`${webappUrl}account/notifications`}
-            >
-              notification settings
-            </a>{' '}
-            can be done anytime through your account details
-          </>
-        ) : (
-          message
+      <div className="mt-2 flex justify-between gap-2">
+        <p
+          className={classNames(
+            'w-full text-theme-label-tertiary tablet:w-3/5',
+            source === NotificationPromptSource.SourceSubscribe && 'flex-1',
+          )}
+        >
+          {acceptedJustNow ? (
+            <>
+              Changing your{' '}
+              <a
+                className="underline hover:no-underline"
+                href={`${webappUrl}account/notifications`}
+              >
+                notification settings
+              </a>{' '}
+              can be done anytime through your account details
+            </>
+          ) : (
+            message
+          )}
+        </p>
+        {source === NotificationPromptSource.SourceSubscribe && (
```

Comment:
Not urgent, but can we not just have one generic image placement?
(I mean would it be acceptable to have the other types also render like this?

Feels kinda dumb to have different image position only for 1 of the types.

Reply:
Yeah, agree, but wanted to get it out of the way. Let me take another stab.

Reply:
Not sure how it looks on desktop, if ok I'll see with others if we can go with it.

Reply:
Yeah non blocking really, also fine if we add a cleanup task somewhere :)

Reply:
@rebelchris since default is absolute position I managed to move it around and have both positions. Still have a bit condition for source but its just on `className`. Let me know what you think.

Reply:
I checked all usages of component and it works.